src/Entity/UserListDefinitionEnum.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Entity;
+
+use MyCLabs\Enum\Enum;
+
+final class UserListDefinitionEnum extends Enum
+{
+    public const TYPE_ELECTED_REPRESENTATIVE = 'elected_representative';
+
+    public const LABEL_ELECTED_REPRESENTATIVE_SUPPORTING_LA_REM = 'Sympathisant(e) LaREM';
```

Comment:
I think we don't need to have a const for the label as each label will be managed by admin